airflow-core/src/airflow/migrations/versions/0071_3_1_0_add_is_active_to_dag_version.py

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Add soft-delete flag to DagVersion (is_active).
```

Comment:
```suggestion
Add soft-delete flag `is_active` to DagVersion.
```

not sure whether it's better this way. it took me a few moment to comprehend the original one

Reply:
Sure. Changed it.

Reply:
The change seems to be missing somehow 🤔

Reply:
That commit got missed to be pushed.

Reply:
Pushed it now